plugin/completion.py

```diff
@@ -147,6 +202,18 @@ def initialize(self) -> None:
             # This is to make ST match with labels that have a weird prefix like a space character.
             settings.set("auto_complete_preserve_order", "none")
 
+    def on_post_text_command(self, command: str, args: dict) -> None:
+        if not self.view.is_popup_visible():
+            return
+        if command in ["hide_auto_complete", "move"]:
+            # hide the popup when `esc` or arrows are pressed pressed
+            self.view.hide_popup()
+
+    def on_modified_async(self) -> None:
+        if self.view.is_popup_visible():
+            # hide the docs popup on each keystoke
+            self.view.hide_popup()
```

Comment:
Does this logic interfere with the signature help logic? It also has an `on_modified_async` callback.

Reply:
Yes, it interferes. 
I did it because when the AC is gone while for example deleting text, the cooperate popup is (sometimes) still visible. I should find a different way to handle this.

Reply:
73988f365dac94ba0fa25ccafea948c5d0dcaa3c